internal/cmd/controller/reconciler/bundle_controller.go

```diff
@@ -215,6 +220,18 @@ func (r *BundleReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctr
 
 	matchedTargets, err := r.Builder.Targets(ctx, bundle, manifestID)
```

Comment:
Do we want to take the opportunity to remove `RetryOnConflict` loops (around L150) for adding and removing finalizers here, as done for other release branches?

Reply:
Is that like a one liner? Otherwise it sounds to me like a different issue/PR.